subsys/nrf_security/src/drivers/cracen/sxsymcrypt/src/platform/baremetal/interrupts.c

```diff
@@ -99,3 +102,42 @@ uint32_t cracen_wait_for_pke_interrupt(void)
 {
 	return cracen_wait_for_interrupt(&cracen_irq_event_for_pke);
 }
+
+#else
+
+void cracen_interrupts_init(void){}
+
+uint32_t cracen_wait_for_cm_interrupt(void)
+{
+	if (k_is_pre_kernel()) {
+		/* Scheduling is not available pre-kernel. Initialization may
+		 * run PRNG at this point.
+		 */
+		return 0;
+	}
+
+	while(!nrf_cracen_event_check(NRF_CRACEN, NRF_CRACEN_EVENT_CRYPTOMASTER)) {
+		/* Clear interrupts in CryptoMaster. */
+		sx_wrreg(REG_INT_STATCLR, ~0);
+
+		/* Clear Cracen wrapper interrupts.*/
+		nrf_cracen_event_clear(NRF_CRACEN, NRF_CRACEN_EVENT_CRYPTOMASTER);
+	}
+
+	return 1;
```

Comment:
I think this should be return 0 ?

Reply:
I think that now this is not relevant since I rewrited it, right?